### ISSUE TITLE
Error on "PApplet" message

### ISSUE THREAD

**Issue by shiffman** [processing/processing#3278]
Picking up on #3253, the following code snippet:

```
void setup() {
  float w = random(20, 300);
  float x = random(w, width-w);
  float y = random(w/2, height-w/2);
  text(int(x) + ","  int(y), 50, 50);  // missing + between "," and int(y)
}
```

produces:
```
Error on "PApplet"`
```

I believe it should say:

```
Error on "int(y)"
```

(unless it can detect something more specific to the error itself.)

**Comment by Manindra29** [comment]
This occurs because presently there's no way for the eclipse compiler to differentiate between:
`int(4.0)` and `PApplet.parseInt(4.0)`. All 'int()' are replaced with `PApplet.parseInt()` during the preprocessing stage, and at the moment, no book-keeping is done to keep track of which parseInt calls replaced int() calls. 

Tldr; it's one of the corner cases for error checking. If this is a relatively common error? Then I can look into how to best handle it.

**Comment by shiffman** [comment]
Ah, I see it's probably of an edge case but in theory it will pop up anytime there's an error at `int(__)`. Do we have the same issues with errors around `color()`?  My concern here is the use of `PApplet` which will cause confusion. So two ideas are:

* Grab the token before `PApplet.parseInt()`.  In this case it would then say `Error on ","`.
* Just change it to say `Unknown error` (or something like that).
